src/validation/rules/ValuesOfCorrectTypeRule.ts

```diff
@@ -221,7 +224,7 @@ function validateOneOfInputObject(
     if (isNullableVariable) {
       context.reportError(
         new GraphQLError(
-          `Variable "$${variableName}" must be non-nullable to be used for OneOf Input Object "${type}".`,
+          `Variable "$${variableName}" must be non-nullable to be used for Within OneOf Input Object Type "${type}".`,
```

Comment:
```suggestion
          `Variable "$${variableName}" must be non-nullable to be used for within OneOf input object type "${type}".`,
```

Reply:
we can introduce a separate PR to get consistency across the code base on this, as well as probably consistency of whether we quote the type/schema element "name", but that should be in a separate non OneOf specific PR

Reply:
https://github.com/graphql/graphql-spec/issues/1112

Reply:
https://github.com/graphql/graphql-js/issues/4196